atlasdb-dbkvs-hikari/src/main/java/com/palantir/nexus/db/pool/HikariCPConnectionManager.java

```diff
@@ -55,6 +55,7 @@
 
     private final ConnectionConfig connConfig;
     private final HikariConfig hikariConfig;
+    private final boolean closeable;
```

Comment:
this is fine, though I think we should probably have some kind of wrapper indicating ownership (see `Ownable<T>`)? fine for testing, maybe not ideal for prod

if bandwidth allows we should get some pairing time tomorrow